README.md

```diff
@@ -3,128 +3,95 @@
 [![CircleCI](https://circleci.com/gh/tulibraries/funcake_dags.svg?style=svg)](https://circleci.com/gh/tulibraries/funcake_dags)
 ![pylint Score](https://mperlet.github.io/pybadge/badges/9.47.svg)
 
-This is the repository for Funnel Cake (PA Digital / DPLA Data QA Interface) Airflow DAGs (Directed Acyclic Graphs, e.g., data processing workflows) for data indexing to Solr + related jobs. These DAGs are expecting to be run within an Airflow installation akin to the one built by our [TUL Airflow Playbook (private repository)](https://github.com/tulibraries/ansible-playbook-airflow).
+This is the repository for Funnel Cake (PA Digital / DPLA Data QA Interface) Airflow DAGs (Directed Acyclic Graphs, e.g., data processing workflows) for data indexing to Solr and related jobs. These DAGs are expecting to be run within an Airflow installation akin to the one built by our [TUL Airflow Playbook (private repository)](https://github.com/tulibraries/ansible-playbook-airflow).
 
-## Repository Structure
 
-This repository has 3 main groups of files:
-- Airflow DAG definition python files (ending with `_dag.py`);
-- Airflow DAG tasks python files used by the above (starting with `task_`);
-- and required local development, test, deployment, and CI files (`tests`, `configs`, `.travis`, Pipfile, etc.).
-
-## Airflow Expectations
-
-These the Airflow expectations for these Funnel Cake DAGs to successfully run:
+## Prequisites
 
 **Libraries & Packages**
 
-- Python 3.7
-- Python Packages: see the [Pipfile](Pipfile)
+- Python: Version as specified in `.python-version`, currently 3.6.8.
+- Pip: 18.1+
+- Pipenv: 2020-11-15+
+- Python Package Dependencies: see the [Pipfile](Pipfile)
+- Docker 20.10+
+- Docker-Compose: 1.27+
 
 **Airflow Variables**
 
+These variable are initially set in the `variables.json` file.
```

Comment:
```suggestion
These variable are initially set in the `variables.json` file.  Variables for this project are primarily handled by the PA Digital team.  They will be the ones to add or update variables as needed.
```

Reply:
This could be helpful information for a new developer on our team to understand the workflow and who manages this.  Is there a reason not to include this?

Reply:
I attempted to incorporate them, but GitHub feature didn't incorporate them in.  Will try again.

Reply:
@sensei100 It looks like they made it in.  Please double check for me..